Sources/TelemetryClient/Signal.swift

```diff
@@ -77,8 +78,16 @@ extension SignalPayload {
         #endif
     }
 
+    static var isDebug: Bool {
+        #if DEBUG
+            return true
+        #else
+            return false
+        #endif
+    }
+
     static var isTestFlight: Bool {
-        guard let path = Bundle.main.appStoreReceiptURL?.path else {
+        guard !isDebug, let path = Bundle.main.appStoreReceiptURL?.path else {
```

Comment:
re-iterating from general PR comment: It is possible (but probably rare) that a developer would make a testflight build in a Debug configuration (by changing the Archive setting in the scheme from Release to Debug). This would defeat the debug/testflight detection implemented in this patch.

Reply:
This is smart! I like it!